swagger-spec/preprints/detail.yaml

```
@@ -47,55 +52,87 @@ get:
                   related:
                     href: https://api.osf.io/v2/preprints/khbvy/identifiers/
                     meta: {}
-              primary_file:
+              node:
                 links:
                   related:
-                    href: https://api.osf.io/v2/files/57c44b1e594d90004a421ab1/
+                    href: https://api.osf.io/v2/nodes/bnzx5/
                     meta: {}
+                data:
+                  id: slug1
+                  type: preprint-provider
```

Seems like there's a bit of copy-paste here.
```suggestion
                    href: https://api.osf.io/v2/nodes/<node_id>/
                    meta: {}
                data:
                  id: "<node_id>"
                  type: "nodes"
```